smartmin/views.py

```diff
@@ -352,12 +352,12 @@ def get_context_data(self, **kwargs):
         # are used to build pagination URLs
         url_params = "?"
         order_params = ""
-        for key in self.request.REQUEST.keys():
```

Comment:
REQUEST doesn't exist in 1.9 as it's now considered bad practice to treat GET/POST variables the same